### DISCUSSION TITLE
Website content: What’s WCAG got to do with it?

### DISCUSSION THREAD

**Discussion by amycole501** [uswds/uswds#5822] [Accessibility]
A site with accessible code becomes inaccessible when content is confusing or too complex. Inaccessible content affects everyone, and this includes English language learners, people with disabilities, and assistive technology users. (You can hear more about the needs of screen reader users on our [May 2023 monthly call recording](https://digital.gov/event/2023/05/18/uswds-monthly-call-may-2023/).)

Using[ plain language techniques](https://www.plainlanguage.gov/) and conforming to [Readable Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/WCAG21/Understanding/readable) can help. Some of the WCAG success criteria related to clear writing include:

- [Reading level](https://www.w3.org/WAI/WCAG21/Understanding/reading-level.html)
- [Unusual words](https://www.w3.org/WAI/WCAG21/Understanding/unusual-words)
- [Abbreviations](https://www.w3.org/WAI/WCAG21/Understanding/abbreviations)

USWDS guidance on plain language and content can be found on many pages of our site. Some examples include:

- The [establishing trust pattern](https://designsystem.digital.gov/patterns/complete-a-complex-form/establish-trust/)
- The [404 page template](https://designsystem.digital.gov/templates/404-page/)
- The [link component](https://designsystem.digital.gov/patterns/complete-a-complex-form/establish-trust/)

We want to hear from you: How do you ensure plain language on your site? What can USWDS do to support teams working towards more accessible content?

**Comment by sarah-sch** [comment]
Becoming aware of conversations around plain language and accessible content is a great place to start. For example, join the [plain language community](https://digital.gov/communities/plain-language/) on [Digital.gov](https://digital.gov/). The listserv will give you a place to ask questions and will show you conversations about challenges that others are facing. You'll also be informed of upcoming learning events.

**Comment by mgifford** [comment]
I do really think that there is value in running text through a tool like [Hemingway App](https://hemingwayapp.com ) 

Worth noting that this is recommended in the [California Design System](https://designsystem.webstandards.ca.gov/style/content/be-concise/)

Also, I really find that [Edge's Reader Mode](https://support.microsoft.com/en-us/topic/use-immersive-reader-in-microsoft-edge-78a7a17d-52e1-47ee-b0ac-eff8539015e1) is great. Nobody likes to read their own work aloud, but it is useful to hear it spoken. Having editors hear their work, can give them some experience of how people will be experiencing it with other text to speech tools, too. I think this is a good practice for authors.

**Reply by amycole501** [reply]
I've used Hemingway App for years and it's been a great tool, especially when introducing people into plain language writing. I'll need to try the reader mode; I've sometimes read my own newsletter or social media posts aloud and it's amazing what changes you're prompted to make when listening versus reading silently. Good tips!
